draft-jholland-quic-multicast.md

```diff
@@ -86,6 +87,7 @@ Thus, this specification has several design goals:
  - Provide flow control and congestion control mechanisms that work with multicast traffic
  - Maintain the confidentiality, integrity, and authentication guarantees of QUIC as appropriate for multicast traffic, fully meeting the security goals described in {{I-D.draft-krose-multicast-security}}
  - Leverage the scalability of multicast IP for data that is transmitted identically to many clients
+ - Rely on Multipath QUIC ({{I-D.draft-ietf-quic-multipath}}) to provide multicast as a service for clients
```

Comment:
I think the implications of keeping this aren't all super clear to me. Right now, the idea is to mostly be transparent to the application layer about multicast. They simply enable the transport parameter and from that point have no longer any feedback on wether multicast works, if any individual packet was received via multi or unicast etc. I am not sure that qualifies as a service or not but I am somewhat cautious of the implications. I think keeping the sentence as is but removing "as a service" would work better for now until we are clear on wether or not it actually is a service for clients or not.

Reply:
I agree. I changed that in the last commit. Thanks.